README.md

```diff
@@ -61,7 +61,7 @@ In which case the resulting message will be:
 ```
 name1=value1 name2=value2
 ```
-For a brief introduction to configuring the syslog-ng application, see the [quickstart guide](https://syslog-ng.com/documents/html/syslog-ng-ose-latest-guides/en/syslog-ng-ose-guide-admin/html/chapter-quickstart.html).
+For a brief introduction to configuring the syslog-ng application, see the [quickstart guide](https://www.syslog-ng.com/technical-documents/list/syslog-ng-open-source-edition/).
```

Comment:
I could not find a link to the non-versioned latest guide, which could be linked directly to the quickstart section.
@fekete-robert Do you know about such link?

We should probably rephrase this sentence if we can't provide a link directly to the quickstart guide.

Reply:
@fekete-robert Thank you!